components/StoryCards/HighlightedStoryCard.module.scss

```diff
@@ -70,3 +70,9 @@
 .link {
     text-decoration: none;
 }
+
+.categoryLink {
+    &:active {
+        color: var(--prezly-header-link-color);
```

Comment:
It feels weird that regular cards have interaction effect on category links, while the highlighted card does not 🤔